app/views/juntos_bootstrap/admin/channels/_form.html.slim

```diff
@@ -13,9 +13,15 @@
       = form.input :permalink, as: :string, required: true
   .w-row
     .w-col.w-col-6
-      = form.input :category_id, as: :select, class: 'medium',
-        collection: Category.order(:name_pt),
-        prompt: t('simple_form.prompts.project.category')
+      = form.input :name,
```

Comment:
Creio que isso ta errado. E pelo que conversamos só rola fazer essa tarefa depois que 
a de back for feita.
Quando for fazer de verdade, acho que rola de fazer uma partial, o que acha?
Já que esse código aqui ta igualzinho ao do outro arquivo.